### ISSUE TITLE
(PS3) Load state doesn't work

### ISSUE THREAD

**Issue by Ezio-PS** [libretro/prosystem-libretro#12]
Load state fails to load the saved states on PS3.

**Comment by Alcaro** [comment]
#11 means savestates are not portable between PS3 and normal platforms.

But if PS3 can't load savestates from PS3, that's a bug.

**Comment by Ezio-PS** [comment]
Yes, it's a bug. A guy reported it on a forum some time ago and i tested it just now. Creating a save state direclty on PS3 is okay, but once i tried to load it i get the yellow error message "Failed to load save state from ....(directory)...."

**Comment by GregorR** [comment]
Loading states doesn't work, period. Breaks netplay too. Either the core shouldn't report that saving states works, or loading them should work.
